GitUI/CommandsDialogs/SettingsDialog/Pages/AdvancedSettingsPage.cs

```diff
@@ -19,6 +19,7 @@ protected override void SettingsToPage()
             chkAlwaysShowAdvOpt.Checked = AppSettings.AlwaysShowAdvOpt;
             chkCheckForRCVersions.Checked = AppSettings.CheckForReleaseCandidates;
             chkRememberIgnoreWhiteSpacePreference.Checked = AppSettings.RememberIgnoreWhiteSpacePreference;
+            chkConsoleEmulator.Checked = AppSettings.UseConsoleEmulatorForCommands;
```

Comment:
This setting went under the Advanced settings page. Which of the actions: disabling/enabling this setting is considered as an advanced one and why?